verl/workers/rollout/schemas.py

```diff
@@ -151,6 +153,16 @@ def get_generation_prompt_ids(self, tokenizer: PreTrainedTokenizer) -> list[int]
         else:
             return self.input_ids
 
+    def add_user_message(
+        self,
+        tokenizer: PreTrainedTokenizer,
+        content: str,
+    ) -> None:
```

Comment:
Since this is an important class and function, i suggest adding docstring to these APIs

Reply:
I will open a issue to track this https://github.com/volcengine/verl/issues/2129.